src/styles/ui.css.js

```diff
@@ -359,4 +376,11 @@ export default css`
   margin: 0 0.5em 0.5em;
   border-bottom-width: 1px;
 }
+
+.respec-hint {
+  display: inline-block;
+  padding-top: .2cm;
+  padding-left: .7cm;
```

Comment:
Oddly specific.

Reply:
heh, yeah... this whole thing needs a rewrite (maybe I'll do that over the break)... I figure, we are already doing `cm` here, so why not.